DEPS

```diff
@@ -68,6 +68,10 @@ Lyrics from lyrics.wikia.com (lyricwiki):
 
 * Beautifulsoup4 (python3)
 
+Modplug file support:
+
+* libmodplug
```

Comment:
Wouldn't this already be required by the GStreamer plugin?

Reply:
You're right, it's in `gstreamer1-plugins-bad-free-extras`. I've removed this change from the PR.